### ISSUE TITLE
SerialPort is at 1.3.1

### ISSUE THREAD

**Issue by nilbus** [PrintToPeer/celluloid-io-serialport#1]
I have no specific need for the newer version, but I wanted to raise a concern, as I had to downgrade my serialport gem version to use celluloid-io-serialport.

**Comment by KazW** [comment]
This was a total experiment on my part, did you ever get this gem working?

I've had the idea of using a serial port from a SideKiq worker recently, so this would help. I'll move this gem to a repo I'm able to maintain and reference this issue so there's a link to the new repo.

**Comment by nilbus** [comment]
Yes, I believe it's working. I don't know if there are any edge cases that don't work properly, but my app instantiates a `Celluloid::IO::SerialPort` and uses it in (almost) the same way as I used to use `::SerialPort` (the difference just being the constructor, IIRC).

If you still have permissions (or can ask the right people), then moving the repository would be an option easier on all the users of this gem, since that way the old URLs will redirect to the new repository location.

**Comment by KazW** [comment]
I'll contact them, but I can't make any promises. However, I still own the gem on RubyGems and can push update through that channel. (including updating repo links :smile:)

**Comment by nilbus** [comment]
`Celluloid::IO` is useful for allowing other tasks within the same actor to be scheduled during blocking IO. Thinking about my usage more deeply, in my current design, I have no such concurrent tasks within the same actor, so I really don't know that this gem is doing its job properly (or that it's not). That said, I think I may soon, so I'll let you know what I find.
